From 1c659d5cc6830c6f4f26660e9049582afbad3fd3 Mon Sep 17 00:00:00 2001
From: Jim Meyering <jim@meyering.net>
Date: Sun, 6 Jan 2013 11:37:15 +0100
Subject: [PATCH 52/89] maint: avoid new warning/error with gcc-4.8.0 20130105

* configure.ac (WERROR_CFLAGS): Disable -Wsuggest-attribute=format.
It was suggesting to apply the gnu_printf attribute to vsnprintf.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index fef0b53..b04eb3f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -235,6 +235,7 @@ if test "$gl_gcc_warnings" = yes; then
   nw="$nw -Wstrict-overflow"        # expr.c, pr.c, tr.c, factor.c
   nw="$nw -Wstack-protector"        # libparted/label/gpt.c
   # ?? -Wstrict-overflow
+  nw="$nw -Wsuggest-attribute=format" # suggests to use gnu_printf for vsnprintf
 
   gl_MANYWARN_ALL_GCC([ws])
   gl_MANYWARN_COMPLEMENT([ws], [$ws], [$nw])
-- 
1.8.5.3

